test/lib/ufe/testGroupCmd.py

```diff
@@ -146,8 +195,11 @@ def testUsdGroup(self):
         self.assertTrue(newGroupPath not in childPathsUndo)
         self.assertTrue(ball5Path in childPathsUndo)
         self.assertTrue(ball3Path in childPathsUndo)
-
-        groupCmd.redo()
+        
+        if ufeUtils.ufeFeatureSetVersion() >= 3:
```

Comment:
Line 180 (for undo) has a different test --- would be nice to pick one way or the other.

Reply:
Not sure if I follow. What is the different test? 

Line 180 is the UFE version check.  

```
if (os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') > '3000'):
        cmds.undo()
    else:
        groupCmd.undo()
```

Reply:
Right, and line 199 is using a ufeFeatureSetVersion() >= 3 for redo.  Seems strange to do one test for undo, and a different one for redo.

Reply:
@ppt-adsk  Je vois!!!! je suis desole. copy paste damage on my part  :)